packages/components/src/alignment-matrix-control/test/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { render, screen, within } from '@testing-library/react';
+import { act, render, screen, within } from '@testing-library/react';
```

Comment:
Adding `waitFor` and removing `act` as per the suggestion in the following comment.

```suggestion
import { render, screen, waitFor, within } from '@testing-library/react';
```